tasks/main.yml

```diff
@@ -96,18 +113,43 @@
     force_basic_auth: true
   register: grafana_conf_existing_dashboards
 
-- name: Find available dashboards
+- name: Validate dashboard repo configuration
+  set_fact:
+    grafana_conf_dashboard_repo_set: "{{
+      (grafana_conf_grafana_dashboard_repo.repo is not none) and
+      (grafana_conf_grafana_dashboard_repo.checkout_path is not none) and
+      (grafana_conf_grafana_dashboard_repo.relative_path is not none) }}"
```

Comment:
Doesn't this break with the role default, an empty dict?

Reply:
Argh :( Thanks!